### ISSUE TITLE
UndefinedMetricWarning: F-score is ill-defined

### ISSUE THREAD

**Issue by cancoo** [howl-anderson/WeatherBot_NLU#4]
anaconda3/envs/weather/lib/python3.6/site-packages/sklearn/metrics/classification.py:1135: UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 in labels with no predicted samples.
  'precision', 'predicted', average, warn_for)

大神，我再train自己数据的时候，最后报上面这个信息。
请教下，这个是什么原因啊？

我看sklearn的版本是  sklearn==0.0

**Comment by howl-anderson** [comment]
sklearn 版本和这个没有关系，F-score 的问题 可以参考 https://stackoverflow.com/questions/43162506/undefinedmetricwarning-f-score-is-ill-defined-and-being-set-to-0-0-in-labels-wi?rq=1 小数据集可能会出现，不是错误因此没什么影响

**Comment by cancoo** [comment]
哦哦，感谢！
some labels in y_true don't appear in y_pred. This means that there is no F-score to calculate for this label, and thus the F-score for this case is considered to be 0.0.
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html
